.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,18 @@
+name: Release Latest Documentation
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
```

Comment:
rm

Reply:
And use https://stackoverflow.com/questions/59518658/how-to-use-github-release-version-number-in-github-action to get the release tag name